### ISSUE TITLE
Allow aliases in optimizer paths

### ISSUE THREAD

**Issue by dgsiegel** [spacecatninja/craft-imager-x#165]
### I'm submitting a... 

- [x] bug report
- [ ] feature request
- [ ] general question

### Steps to reproduce

1. Put an optimizer in a relative directory (e.g. `../bin/jpegoptim`)
2. Try to use an alias to refer to that directory (e.g. `'path' => '@root/../bin/jpegoptim'`)
3. Optimization will fail. (`Optimizer spacecatninja\imagerx\optimizers\JpegoptimOptimizer could not be found in path @root/../bin/jpegoptim`)

### Description

This is the relevant part of `config/imager-x.php`:

```
  'optimizers' => ['jpegoptim'],
  'optimizerConfig' => [
    'jpegoptim' => [
      'extensions' => ['jpg'],
      'path' => '@root/../bin/jpegoptim',
      'optionString' => '--max=80 --strip-all',
    ],
  ]
```

I've also tried a relative path, but that fails as well:

```
  'optimizers' => ['jpegoptim'],
  'optimizerConfig' => [
    'jpegoptim' => [
      'extensions' => ['jpg'],
      'path' => '../bin/jpegoptim',
      'optionString' => '--max=80 --strip-all',
    ],
  ]
```

Interestingly, using `'/../bin/jpegoptim` does seem to work:

`2022-09-08 13:45:02 [-][1][-][info][application] Command "/../bin/jpegoptim --max=80 --strip-all "~web/htdocs/media/images/114312/foo_ed0a4c0106.jpg"" returned "~web/htdocs/media/images/foo_ed0a4c0106.jpg 1920x1920 24bit N JFIF  [OK] 396315 --> 396067 bytes (0.06%), optimized."`

Also, the following will work:

```
  'optimizers' => ['jpegoptim'],
  'optimizerConfig' => [
    'jpegoptim' => [
      'extensions' => ['jpg'],
      'path' =>  CRAFT_BASE_PATH . '/../bin/jpegoptim',
      'optionString' => '--max=80 --strip-all',
    ],
  ]

```

All in all it seems that `php-shellcommand` requires an absolute path.

**Comment by aelvan** [comment]
Support for aliases and environment variables in path and various other optimizer settings has been added to 4.1.3 and 3.6.3.

**Comment by dgsiegel** [comment]
I think you've missed `file_exists(App::parseEnv($settings['path'])` in the v3 branch :) If you compare https://github.com/spacecatninja/craft-imager-x/commit/e5960bfef37f54d93a4bfa26b235a205d8b7a250 with https://github.com/spacecatninja/craft-imager-x/commit/e10185ad5f8bd99b75a80d7fb6eefaaf961c208a you can see that this check is correct in the v4 branch/master.

**Comment by aelvan** [comment]
Thank you! Fixed in 3.6.3.1.
